Convert impersonal "users" to direct "you" in documentation (#4959)
ClickHouse/clickhouse-docs: docs/cloud/features/04_infrastructure/automatic_scaling/01_auto_scaling.md

<ScalePlanFeatureBadge feature="Automatic vertical scaling"/>

:::note

> Scale and Enterprise tiers supports both single and multi-replica services, whereas, Basic tier supports only single replica services. Single replica services are meant to be fixed in size and do not allow vertical or horizontal scaling. You can upgrade to the Scale or Enterprise tier to scale your services.

Reviewer: ```suggestion
Scale and Enterprise tiers supports both single and multi-replica services, whereas, Basic tier supports only single replica services. Single replica services are meant to be fixed in size and don't allow vertical or horizontal scaling. You can upgrade to the Scale or Enterprise tier to scale your services.
```
